<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>549</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>549</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="LoadFileButton">
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Load file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaveFileButton">
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Save file</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_17">
      <item row="7" column="1">
       <widget class="QPushButton" name="LogFileButton">
        <property name="text">
         <string>Log file</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QTextEdit" name="TdelayText">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>29</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QTextEdit" name="FSampleText">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>29</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="TCFileButton">
        <property name="text">
         <string>TC file</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="TMFileButton">
        <property name="text">
         <string>TM file</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QTextEdit" name="LogFileText">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>29</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QTextEdit" name="TCFileText">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>29</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QTextEdit" name="TMFileText">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>29</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>29</height>
         </size>
        </property>
        <property name="text">
         <string>Tdelay</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_1">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>29</height>
         </size>
        </property>
        <property name="text">
         <string>FSample</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>29</height>
         </size>
        </property>
        <property name="text">
         <string>I1 Threshold</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>29</height>
         </size>
        </property>
        <property name="text">
         <string>I2 Threshold</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QTextEdit" name="I2Text">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>29</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QTextEdit" name="I1Text">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>29</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="DefaultsButton">
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Defaults</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="OKButton">
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>785</width>
        <height>228</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
